test/remote/gateways/remote_aafes_test.rb

```diff
@@ -0,0 +1,167 @@
+require 'test_helper'
+
+class RemoteAafesTest < Test::Unit::TestCase
+  def setup
+    @gateway = AafesGateway.new(fixtures(:aafes))
+    @gateway.ssl_strict = false # Fails to send request to gateway otherwise :shrug
+    @amount = '%.2f' % 100
+    @metadata = {
+      :zip => 75236,
+      :expiration => 2210
+    }
+    
+    @milstar_card = ActiveMerchant::Billing::PaymentToken.new(
+      '900PRPYIGCWDS4O2615',
+      @metadata
+    )
```

Comment:
I want to point this out, in case this is the incorrect use of `PaymentToken`.

Reply:
It doesn't look like there are any other examples of `PaymentToken` getting used as a top-level citizen; I wonder if we need to make a fresh token type?